web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -23,7 +23,7 @@ export const getIsApplicableToFunctionByFieldName = (
   return (industryId: string | undefined): boolean => {
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     return EssentialQuestions.find((eQ) => {
-      return eQ.fieldName == fieldName || eQ.contentFieldName == fieldName;
+      return eQ.fieldName == fieldName;
```

Comment:
```suggestion
      return eQ.fieldName === fieldName;
```
Another one of those techtro items from last week.